hack/verify-testing-import.sh

```diff
@@ -40,21 +41,23 @@ RELEASE_BIN_PKGS=(
   "${KUBE_ROOT}/cmd/kubectl-convert"
   "${KUBE_ROOT}/cmd/kubelet"
   "${KUBE_ROOT}/cmd/kubeadm"
```

Comment:
I didn't use `"${KUBE_ROOT}"/cmd/*/ in the final PR because:
- it actually doubles the time to run this (we have a lot of small utilities we don't release)
- it will fail on kubemark
- the utility seemed low